Guidelines.md

```diff
@@ -1006,6 +1007,27 @@ When these operations are performed together, the evaluation order MUST be:
 2. **Sorting**. The potentially filtered list is sorted according to the sort criteria.
 3. **Pagination**. The materialized paginated view is presented over the filtered, sorted list. This applies to both server-driven pagination and client-driven pagination.
 
+### 9.10. Empty Results
+When a search is performed on a collection and the result set is empty you MUST respond with a valid response body and a 200 response code. 
```

Comment:
Please don't refer to this as a search as that's confusing with an actual $search operation which is usually very different from a filtered query on a collection.

Reply:
how about the word filter instead of search?